### ISSUE TITLE
Locking up on startup

### ISSUE THREAD

**Issue by vangalvin** [madhephaestus/Esp32WifiManager#5]
Things seem to stop at "starting wifi manager"

On its own everyting seems to work great, really fast connection times and easy to alter the system however after adding the routines to get the value of the sensors etc connections seem to be a bit hit and miss and im not exactly sure as to why.

If i reload the sketch several times it springs back in to life but only sometimes.

Any sugestions?

**Comment by madhephaestus** [comment]
do you have your project posted on github to take a look?

**Comment by madhephaestus** [comment]
I can't reproduce, closing. Please attach a project that demonstrates the error if you would like to reopen. I suspect hardware issues.

**Comment by vangalvin** [comment]
@madhephaestus i meant to post an update to this yesterday but got tied up.

I found the issue...
With my system if I am not running WiFi the system just reads the pressure sensors and spits out the data to serial and the LCD. So for that i had added Serial.begin(115200); at the start of setup()

The lib for ESP32WiFiManager also has a Serial.begin(115200); so when Serial is already started and the WiFiManager starts up it is doing a Serial.begin as well resulting in it locking up.

Removed one of them and its all working perfectly :)

Thanks for the response
